src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -22,13 +22,15 @@ def initialize(disks, pager)
 
       # table items. See CWM::Table#items
       def items
+        probed_graph = Y2Storage::StorageManager.instance.y2storage_probed
```

Comment:
so you suggest that every single widget is created with "old_graph" and "new_graph" as parameters? To be honest in past there is formatted flag in target_map, which I find even better solution then this one, where you basically have to have everywhere "old target map" and "new target map" ( replace map with graph ) and have to call method to detect change between them on so called object, which is in fact just pointer to graph with few methods around.